clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -683,7 +684,8 @@ Expected<StringRef> compileModule(Module &M, OffloadKind Kind) {
 /// registration code from the device images stored in \p Images.
 Expected<StringRef>
 wrapDeviceImages(ArrayRef<std::unique_ptr<MemoryBuffer>> Buffers,
-                 const ArgList &Args, OffloadKind Kind) {
+                 ArrayRef<OffloadingImage> Images, const ArgList &Args,
```

Comment:
It looks like this Images parameter is leftover from and older change

Reply:
Oh, yes. Good catch.